north/src/runtime/error.rs

```diff
@@ -116,8 +116,8 @@ pub enum KeyError {
 }
 
 #[derive(Error, Debug)]
-pub enum InstallFailure {
-    #[error("File seems to be corrupted")]
+pub enum Installation {
```

Comment:
I generally agree that elements in an enum that's called XYZError do not need the Error, but a top level enum? I don't think that `Installation` or `Key` is the right term to describe an `InstallationError` or `KeyProblem`

Reply:
I think the additional `Error` can be omitted because they all are in the mod `error`. This pattern is also followed in the stdlib. e.g here: https://doc.rust-lang.org/std/io/enum.ErrorKind.html

I would also strongly suggest to stick to one term: Error or Problem.

Reply:
In ErrorKind the error types are self-descriptive, that's fine.
![image](https://user-images.githubusercontent.com/99541/97960535-e991de80-1db1-11eb-91c7-98171f87bceb.png)
Calling an enum `Key` does not reflect in any way that it actually is an error...even if it currently resides in the `error` module and actually is misleading.